.pre-commit-config.yaml

```diff
@@ -10,11 +10,7 @@ repos:
             )$
 
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.750
-    # Currently mypy may assert after logging one message. To get all the messages at once, change repo and rev to
-    #   repo: https://github.com/chriselion/mypy
-    #   rev: 3d0b6164a9487a6c5cf9d144110b86600fd85e25
-    # This is a fork with the assert disabled, although precommit has trouble installing it sometimes.
+    rev: v0.761
```

Comment:
Tried bumping this to get the errors to show, but no luck. Removed the obsolete comment.